### DISCUSSION TITLE
Seeking feedback for removing Pass Keyword, replace it with string or null not assigned to variable

### DISCUSSION THREAD

**Discussion by SilencedPerson** [godotengine/godot-proposals#5603] [Editor]
Instead of the current:
```gdscript
# Called when the node enters the scene tree for the first time.
func _ready():
	pass # Replace with function body.
```
It would be:
```gdscript
func _ready():
	"""Called when the node enters the scene tree for the first time.
	Replace these lines with function body."""
```
or:
```gdscript
# Called when the node enters the scene tree for the first time.
func _ready():
	null # Replace with function body.
```
The string variant is my preferred version of this change, as it also showcases "multiline comments" in Godot along with multiline strings.

**Comment by Zireael07** [comment]
Bad idea. String solution would force users to write pseudo-docs every time they want to "pass" (not to mention I often find myself  having a pass that is commented out depending on needs)

**Reply by CarpenterBlue** [reply]
Nah, I see the point. This is already possible in GDScript.
You are actually not required to write in "pseudo-docs". It's about removing the keyword and changing the default script.
The fastest way to do this is actually:
```gdscript
func example():
    0
```
than writing pass anyway, plus you are not really supposed to leave the empty function in anyway. This change would probably affect only the built in functions and the default script.
```gdscript
func example():
    pass
```

**Comment by mieldepoche** [comment]
> "multiline comments"

there are no multiline comments in godot. `"""..."""` are not comments and don't get stripped on export.

```gdscript
# Called when the node enters the scene tree for the first time.
func _ready():
    null # Replace with function body.
```
You can already do that, if you're talking about the general case (altough this make no sense imo). If you're talking about specifically the default script content I don't get why you would want this changed.
